sdk/core/core/src/az_pal.c

```diff
@@ -0,0 +1,24 @@
+#include <az_pal.h>
+#include <az_time_internal.h>
+
+#ifdef _WIN32
+#include <windows.h>
+#else
+#include <unistd.h>
+#endif
+
+#include <_az_cfg.h>
+
+#ifdef _WIN32
+void az_pal_sleep(int32_t milliseconds) {
+  if (milliseconds > 0) {
+    Sleep(milliseconds);
+  }
+}
+#else
```

Comment:
This only works on Linux. Should the condition be an `#elif` and a `#error` with a clear message that the PAL needs an implementation on other systems?

Reply:
We now have POSIX, Win32, and "empty" PAL. Please wait for the updated PR where I am going to use it.